code/style.css

```diff
@@ -1,7 +1,325 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
 
-.active + .answer {
-  /* some styling here */
+* {
+  padding: 0; 
+  margin: 0; 
 }
+
+body { 
+  background-color: rgb(255, 255, 255);
+  font-family: "Montserrat";
+
+}
+
+a {
+  color: rgb(88, 88, 88);
+  text-decoration: none;
+}
+
+a:hover {
+  color: #00747c;
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+.header { 
+  display: flex;
+  justify-content: center; 
+  width: auto;
+
+  /* border: 3px solid green; */
+}
+
+video { /*  https://css-tricks.com/full-page-background-video-styles/ */
+  object-fit: cover;
+  width: 100%;
+  height: 400px;
+}
+
+h1 { 
+  text-align: center;
+  color: rgb(235, 233, 233);
+  text-shadow:2px 2px #4e4e4e;
+  font-size: 50px;
+  text-transform: uppercase;
+}
+
+h2 {
+  text-align: center;
+  color: rgb(235, 233, 233);
+  text-shadow:2px 2px #4e4e4e;
+  font-size: 30px;
+  text-transform: uppercase;
+  padding: 1rem;
+}
+
+.overlay {
+  position:absolute;
+  top:5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: auto;
+  /* border:2px solid green; */
+}
+
+button {
+  padding:15px;
+  background-color: cadetblue;
+  border:none;
+  border-radius: 8px;
+  font-family: "Montserrat";
+  color:white;
+  cursor: pointer;
+}
+
+button:hover {
+  background: #00747c;
+  transition: 0.25s;
+}
+
+/* MAIN SECTION*/
+.container {
+  display: flex;
+  justify-content: center;
+  width: auto;
+  /* border:4px solid violet; */
+}
+
+/* FORM SECTION */
+
+/* fieldset and legend */ 
+fieldset {
+  margin: 2rem 0 ; 
+  padding: 1em 2em;
+  border: 1px solid rgb(110, 110, 110);
+  border-radius: 6px;
+}
+
+legend {
+  font-size:1.25em;
+  padding:0 .25em;
+  color: #999;
+}
+
+/*labels*/
+label {
+  display: block; /* places all element in a row*/ 
```

Comment:
Smart, wish I thought of this as well instead of using good old `<br>`! 💯

Reply:
I used <br> at first, but then I wanted to add all of the styling in the css instead.